nimbus/src/lib.rs

```diff
@@ -60,19 +60,6 @@ impl NimbusClient {
         })
     }
 
-    // This is a little suspect but it's not clear what the right thing is.
-    // Maybe it's OK we initially start with no experiments and just need to
-    // wait for the app to schedule a regular `update_experiments()` call?
```

Comment:
I think that not only is it *OK* to rely on the app to call `update_experiments`, it's well aligned with the overall philosophy of who should be in charge of dong what in this component.